ui/assets/styles/general/tool.styles.js

```diff
@@ -52,9 +52,15 @@ const useStyles = makeStyles((theme) => ({
     position: 'relative',
     marginTop: '1rem',
     transition: 'all 1s',
+    [theme.breakpoints.down('sm')]: {
```

Comment:
@ptdatta what else uses these styles? Please list here.

For each one, confirm that these changes don't have a negative affect.

Reply:
@ptdatta ?